Cert_6_1_09_EDSynchronization.json

```diff
@@ -0,0 +1,83 @@
+{
```

Comment:
Is adding this file (and pcap file below) intentional?

Reply:
sorry, wrong commit, deleted